addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -1902,6 +1902,44 @@ QUnit.module('basic_fields', {
         assert.verifySteps(['destroy']);
     });
 
+    QUnit.test('graph dashboard widget can be destroyed when nv is partially loaded', function (assert) {
+        // this test ensures that the JournalDashboardGraph widget doesn't crash
+        // when being destroyed before nv has been completely loaded
+        assert.expect(2);
+
+        var destroy = basicFields.JournalDashboardGraph.prototype.destroy;
+        basicFields.JournalDashboardGraph.prototype.destroy = function () {
+            assert.step('destroy');
+            var nv = window.nv;
+            // nv is fully loaded only when nvd3.js has been loaded
+            // which happens sequentially after nv.d3.js
+            // we simulate this race condition with:
+            testUtils.patch(window.nv.utils, {offWindowResize: undefined});
```

Comment:
no need here, what you did was ok, i think

Reply:
>lol, no
how mean :'-(

Reply:
didn't want to, sorry

Reply:
Previous build was incorrect, now fixed.